create-new-data-pr.py

```diff
@@ -35,29 +45,38 @@ def update_tag_version(current_version=None):
   data_prid = int(opts.pull_request)
   dist_repo = gh.get_repo(opts.dist_repo)
   data_repo_pr = data_repo.get_pull(data_prid)
-  data_repo_base_branch = data_repo_pr.base.ref
+  data_pr_base_branch = data_repo_pr.base.ref
+  data_repo_default_branch = data_repo.default_branch
   # create master just to exist on the cms-data repo if it doesn't
-  if data_repo_base_branch == "main":
+  if data_repo_default_branch == "main":
       if "master" not in [branch.name for branch in data_repo.get_branches()]:
-          data_repo.create_git_ref(ref='refs/heads/master', sha=data_repo.get_branch(data_repo_base_branch).commit.sha)
+          data_repo.create_git_ref(ref='refs/heads/master', sha=data_repo.get_branch(data_repo_default_branch).commit.sha)
 
-  if (data_repo_base_branch != "master") and (data_repo_base_branch != "main"):
-      print("I do not know how to tag a non-master (non-main) branch %s" % data_repo_pr.base.ref)
-      exit(1)
+  last_release_tag = None
+  repo_name_only = opts.data_repo.split("/")[1]
+  #find the last tag on the default branch, which is needed in all cases
+  err, out = run_cmd("rm -rf repo && git clone --bare https://github.com/%s -b %s repo && GIT_DIR=repo git log --pretty='%%d'" % (opts.data_repo, data_repo_default_branch))
+  last_release_tag = get_tag_from_string(out)
+
+  if (data_pr_base_branch != data_repo_default_branch):
```

Comment:
remove this whole block and just use `data_pr_base_branch` in the above git clone command. No need to get latest tags for default branch

Reply:
removed